examples/00-test/index.html

```diff
@@ -51,15 +51,17 @@ <h1>
 			<button data-tool="zoom">Zoom</button>
 			<button data-tool="zoomTouchPinch">ZoomTouchPinch</button>
 			<button data-tool="zoomMouseWheel">ZoomMouseWheel</button>
+			<button data-tool="freehand">Freehand</button>
 
 			<div class="canvas" oncontextmenu="return false" />
 		</div>
 	</div>
 </body>
 
-<!-- include the hammer.fakemultitouch so we can test with a mouse -->
+<!-- include the hammer.fakemultitouch so we can test with a mouse
 <script src="../touch-emulator.js"></script>
 <script>TouchEmulator();</script>
+-->
```

Comment:
Hey, yeah I have no touch interface for the freehand tool, so I disabled it for testing. We could add a button to the demo UI to switch it on or off?

Reply:
I think we can safely remove this altogether. The majority of tools now support touch and mouse, so we shouldn't need to emulate touch events to make sure the tools are working. It may even cause issues to attempt emulation.

Can you

Reply:
Done.